### ISSUE TITLE
Update Brassicaceae checklist through GBIF

### ISSUE THREAD

**Issue by chantalhuijbers** [CatalogueOfLife/data#264]
COL has a 2009 version of the Brassicaceae checklist, but GBIF already has a 2021 version in DwC-A available. See: https://www.gbif.org/dataset/94308742-058c-46d5-b763-06e9207a6b15 The authors of the checklist have given us permission to use the checklist in GBIF to update the Brassicaceae checklist in COL.
@yroskov or @gdower can you get this organised?
------------------------------------------
Email from checklist contact:

Hi Chantal,

Thank you for your email. Ardath Francis is a volunteer in our group and since the pandemic she has lost access to her work email, but I would be happy to help as I was the one who prepared the publication of the new dataset.

It is great to hear that you would like to make the new Brassicaceae checklist available in COL. As per your question, the checklist can be licensed under CC BY. For the citation, the new checklist has been published in the Biodiversity Data Journal: https://pubmed.ncbi.nlm.nih.gov/33716543/ .

I would be happy to help if you have any questions or need anything else from me,
Beatriz

**Comment by yroskov** [comment]
That's great. If data format is compatible with the checklistbank, we can update checklist even before AC21. I do not know, whether Geoff has time to assess format or not. Who else can do assessment for CoL?

**Comment by mdoering** [comment]
https://data.catalogueoflife.org/dataset/2305/about

**Comment by yroskov** [comment]
Great, Markus! 

Could you please add family Brassicaceae as a top taxon in the checklist tree. Otherwise, CoL will have 300+ new sectors.

**Comment by yroskov** [comment]
@chantalhuijbers, could you please take care about Brassicaceae metadata for CoL

**Comment by chantalhuijbers** [comment]
Thanks guys! Yes, I can look after getting the metadata. @mdoering probably easiest if I do that immediately for the new metadata schema. What would be the best way to get this? Just send them the yaml file to complete?

**Comment by mdoering** [comment]
The Brassicaceae list is managed by Agriculture and Agri-Food Canada and published by Pensoft via the IPT from their Biodiversity Data Journal: http://ipt.pensoft.net/resource.do?r=aafc-brassicaceae-checklist

It contains a rather rich EML document already. Are we sure their metadata is missing anything?
We should and cannot manage the metadata for external datasets. We can request data providers to provide good metadata, but they have the authority over their dataset and when we change it in CLB it will be overwritten with their copy at some stage anyways.

That's why we have project specific patches to override the original metadata to fit into the COL.
But is this needed in this case? We should migrate the alias from the previous list to the new one. 

One other thing. Should we maybe reuse the previous dataset so we don't change keys? Then the alias would also stay.

**Comment by chantalhuijbers** [comment]
If this metadata is available, I would go with that. If there are specific things that we are missing or are unsure of we can communicate with the GSD contact person.

**Comment by mdoering** [comment]
The updated Brassicaceae was added to COL, but the dataset key has now changed.
@yroskov was this done on purpose or should we revert that change? Keeping the same keys for source datasets is an important aspect of managing the COL checklist. Otherwise:

 - a new DOI is assigned for the source
 - we lose the metrics history and the ability to quickly compare changes between releases
 - the GSD will show as a new GSD
 - and probably more undesired side effects...

**Comment by yroskov** [comment]
@gdower, could you pls have look on this next week?

**Comment by mdoering** [comment]
it appears all accepted infraspecific names have been blocked. That's probably the best solution until the dataset is improved or the [backend can more cleverly deal with unplaced subspecies](https://github.com/CatalogueOfLife/backend/issues/1009).

It might have been a lot simpler to just list the acceptable ranks in the sector, then no decisions would have been needed to exclude subspecific taxa. But it might have blocked also infraspecific synonyms for species which we don't need to exclude.

**Comment by yroskov** [comment]
Only accepted trinomials have been blocked.

**Comment by mdoering** [comment]
Yes, that makes sense. Just saying it would have been a lot simpler with just changing the sector rank filter. Then there is no need for a decision.
